src/applications/personalization/profile-2/tests/components/DirectDeposit.unit.spec.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { mount } from 'enzyme';
+import { expect } from 'chai';
+
+import DowntimeNotification, {
+  externalServices,
+} from 'platform/monitoring/DowntimeNotification';
+
+import DirectDeposit from '../../components/direct-deposit/DirectDeposit';
+import DirectDepositContent from '../../components/direct-deposit/DirectDepositContent';
+
+describe('DirectDeposit', () => {
+  let wrapper;
+  let directDeposit;
+  let fakeStore = {};
+  const makeFakeStore = () => ({
```

Comment:
I'm not really sure how ideal it is, really :-/ It's fine for when you need to mount a Redux-connected component to do some pretty simple tests, but I've found it to be lacking when trying to write some more comprehensive. I want to be able to mount the DirectDepositContent component and click around to interact with the different views it can render. But I can't do that since actions do not get dispatched and the Redux store is not updated so the component's props never change and it never renders anything different.